sw/device/tests/chip_power_idle_load.c

```diff
@@ -0,0 +1,516 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//==============================================================================
+// Included code:
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"  // Memory-mapped IO functions
+#include "sw/device/lib/dif/dif_alert_handler.h"
+#include "sw/device/lib/dif/dif_gpio.h"
+#include "sw/device/lib/dif/dif_otp_ctrl.h"
+#include "sw/device/lib/dif/dif_pinmux.h"
+#include "sw/device/lib/dif/dif_pwm.h"
+#include "sw/device/lib/dif/dif_pwrmgr.h"
+#include "sw/device/lib/dif/dif_rv_core_ibex.h"
+#include "sw/device/lib/dif/dif_rv_timer.h"
+#include "sw/device/lib/runtime/hart.h"  // functions for excution halt-like functionality
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/alert_handler_testutils.h"
+#include "sw/device/lib/testing/aon_timer_testutils.h"
+#include "sw/device/lib/testing/pwrmgr_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_isrs.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+#include "alert_handler_regs.h"  // Generated.
+#include "aon_timer_regs.h"
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"  // Top-specific Definitions
+#include "pwm_regs.h"
+
+//==============================================================================
+// Global Variables
+
+typedef void (*isr_handler)(void);
+static volatile isr_handler expected_isr_handler;
+static volatile dif_rv_core_ibex_nmi_state_t nmi_state;
+static volatile bool nmi_fired = false;
+static volatile bool ext_irq_fired = false;
+static volatile bool irq_is_pending = false;
+
+static dif_aon_timer_t aon_timer;
+static dif_rv_core_ibex_t rv_core_ibex;
+static dif_pwrmgr_t pwrmgr;
+static dif_rv_timer_t rv_timer;
+static dif_alert_handler_t alert_handler;
+static dif_pwm_t pwm;
+static dif_pinmux_t pinmux;
+static dif_otp_ctrl_t otp_ctrl;
+static dif_gpio_t gpio;
+
+//==============================================================================
```

Comment:
Remove guard style comments throughout the file.

Reply:
Done.